src/editing_tools/AdjustTool.vala

```diff
@@ -371,7 +419,8 @@ public class EditingTools.AdjustTool : EditingTool {
 
         /* set up temperature & tint */
         TemperatureTransformation temp_trans = (TemperatureTransformation)
```

Comment:
Or we can use `var` for the left operand, like this:

```vala
var temp_trans = (TemperatureTransformation) transformations.get_transformation (
    PixelTransformationType.TEMPERATURE
);
```

Reply:
I think so too :) Thanks!